platform-tests/src/acceptance/http_edge_cases_test.go

```diff
@@ -0,0 +1,197 @@
+		var appName string
+
+		BeforeEach(func() {
+			appName = generator.PrefixedRandomName("CATS-APP-HTTP-TESTER-")
+			Expect(cf.Cf(
+				"push", appName,
+				"-b", config.GoBuildpackName,
+				"-p", "../../example-apps/http_tester",
+				"-d", config.AppsDomain,
+				"-c", "./bin/http_tester; sleep 1; echo 'done'",
+			).Wait(CF_PUSH_TIMEOUT)).To(Exit(0))
+		})
+
+		It("should accept large request body sizes", func() {
+			for requestkB := 1; requestkB <= 200; requestkB += 10 {
+				By(fmt.Sprintf("request size of %d kB", requestkB))
+				buffer := new(bytes.Buffer)
+				file, err := os.Open("/dev/zero")
+				Expect(err).NotTo(HaveOccurred())
+				defer file.Close()
+
+				copied, err := io.CopyN(buffer, file, int64(requestkB*int(KILOBYTE)))
+				Expect(err).NotTo(HaveOccurred())
+				fmt.Fprintf(GinkgoWriter, "Successfully copied %d bytes\n", copied)
+
+				appUri := helpers.AppUri(appName, "/body-size")
+
+				req, err := http.NewRequest("POST", appUri, buffer)
+				Expect(err).NotTo(HaveOccurred())
+
+				response, err := httpClient.Do(req)
+				Expect(err).NotTo(HaveOccurred())
+				returnedBody, err := ioutil.ReadAll(response.Body)
+				Expect(err).NotTo(HaveOccurred())
+				returnedBodySize, err := strconv.Atoi(string(returnedBody))
+				Expect(err).NotTo(HaveOccurred())
+				Expect(returnedBodySize).Should(Equal(requestkB * int(KILOBYTE)))
+			}
+		})
+
+		It("should accept large request header sizes", func() {
+			for headerkB := 1; headerkB <= 7; headerkB += 1 {
+				By(fmt.Sprintf("request header size of %d kB", headerkB))
+
+				curlArgs := []string{"-H", fmt.Sprintf("test-header: %s", randStringBytes(headerkB*int(KILOBYTE)))}
+				response := helpers.CurlApp(appName, "/header-size", curlArgs...)
+
+				Expect(response).To(BeEquivalentTo(strconv.Itoa(headerkB * int(KILOBYTE))))
+			}
+		})
+
+		It("should accept large response header sizes", func() {
+			for headerkB := 1; headerkB <= 7; headerkB += 1 {
+				By(fmt.Sprintf("response header size of %d kB", headerkB))
+
+				appUri := helpers.AppUri(appName, fmt.Sprintf("/big-header?size=%d", headerkB))
+
+				response, err := httpClient.Get(appUri)
+				Expect(err).NotTo(HaveOccurred())
+				testHeader := response.Header.Get("test-header")
+				Expect(len(testHeader)).To(Equal(headerkB * int(KILOBYTE)))
+			}
+		})
+
+		It("allow egress connectivity", func() {
+			response := helpers.CurlApp(appName, "/egress?domain=www.gov.uk")
+
+			Expect(response).To(BeEquivalentTo("OK"))
+		})
+
+		It("can connect to other app in the paas", func() {
+			appName2 := generator.PrefixedRandomName("CATS-APP-HTTP-TESTER-")
+			Expect(cf.Cf(
+				"push", appName2,
+				"-b", config.GoBuildpackName,
+				"-p", "../../example-apps/http_tester",
+				"-d", config.AppsDomain,
+				"-c", "./bin/http_tester; sleep 1; echo 'done'",
+			).Wait(CF_PUSH_TIMEOUT)).To(Exit(0))
+
+			response := helpers.CurlApp(appName, fmt.Sprintf("/egress?domain=%s.%s", appName2, config.AppsDomain))
```

Comment:
This test fails since we removed `InsecureSkipVerify` from the `/egress` handler in the app:
```
HTTP edge cases
/tmp/build/3151afb2/paas-cf/platform-tests/src/acceptance/http_edge_cases_test.go:188
  Using http_tester
  /tmp/build/3151afb2/paas-cf/platform-tests/src/acceptance/http_edge_cases_test.go:187
    can connect to other app in the paas [It]
    /tmp/build/3151afb2/paas-cf/platform-tests/src/acceptance/http_edge_cases_test.go:145

    Expected
        <string>: Get https://CATS-APP-HTTP-TESTER-3e568d99-f5c0-42a8-7d92-2b3c86db4ea4.dcarley.dev.cloudpipelineapps.digital/: x509: certificate signed by unknown authority
        
    to be equivalent to
        <string>: OK

    /tmp/build/3151afb2/paas-cf/platform-tests/src/acceptance/http_edge_cases_test.go:144
```

I think the options are either:

1. Add a query parameter to disable cert verification for this one test based on `config.SkipSSLValidation`
1. Add our CA to the rootfs for environments that are self-signed